lib/auth/integration/integrationv1/awsra.go

```diff
@@ -224,3 +224,79 @@ func (s *AWSRolesAnywhereService) ListRolesAnywhereProfiles(ctx context.Context,
 		NextPageToken: profileList.NextToken,
 	}, nil
 }
+
+// AWSRolesAnywherePing performs a health check for the AWS Roles Anywhere integration.
+// If the integration is absent from the request, then it will use the trust anchor, profile and role from the request.
+// This is useful for testing an integration that was not yet created.
+//
+// If the integration is present in the request, it will use the trust anchor and the profile, role from the integration's ProfileSync config.
+//
+// It returns the caller identity and the number of AWS Roles Anywhere Profiles that are active.
+func (s *AWSRolesAnywhereService) AWSRolesAnywherePing(ctx context.Context, req *integrationpb.AWSRolesAnywherePingRequest) (*integrationpb.AWSRolesAnywherePingResponse, error) {
+	authCtx, err := s.authorizer.Authorize(ctx)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	if err := authCtx.CheckAccessToKind(types.KindIntegration, types.VerbUse); err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	trustAnchor := req.GetTrustAnchorArn()
+	profileARN := req.GetProfileArn()
+	roleARN := req.GetRoleArn()
+
+	// If integration was provided, it uses the Trust Anchor configured and the Profile/Role configured for ProfileSync.
+	if req.Integration != "" {
```

Comment:
nit: maybe some debug log on what will be tested?

what happens when both integration and the ARNs are specified in the request? (and the other way as well when nothing is specified).

Reply:
Added a debug log 👍 

If nothing is set, then it will fail when parsing the trust anchor arn.
If both are set, then the integration will take precedence.